### PR TITLE
Add clickable breadcrumb trail

### PR DESCRIPTION
**Pull request by taylortom**
A solution for #1486, based on the UI refresh designs.

**TODO** remove extraneous commits

### CODE REVIEW

File: frontend/src/core/dashboard/dashboard.js [hunk 1]
```diff
@@ -7,88 +7,117 @@ define(function(require) {
...
+        var dashboardType = location || 'all';
+        Origin.trigger('dashboard:loaded', { type: dashboardType });
       });
     });
-  });
-  Origin.on('dashboard:loaded', function (options) {
+  }
+
+  function onDashboardLoaded(options) {
+    console.log('onDashboardLoaded', options);
```
Comment (dancgray) [inline frontend/src/core/dashboard/dashboard.js]: Can this console.log be removed please.

**Comment by dancgray** [comment]
Looking good. It's giving warning for missing lang values:
Missing translation for key: "app.editorsettingstitle"
Missing translation for key: "app.editorconfigtitle"
Missing translation for key: "app.editormenusettingstitle"
Missing translation for key: "app.editorextensionstitle"
Missing translation for key: "app.editorpagesettings"
Missing translation for key: "app.editorarticle"
Missing translation for key: "app.editorblock"
Missing translation for key: "app.editorcomponent"

Although only app.editorarticle, app.editorblock, and app.editorcomponent are displayed as app.editorarticle etc.
@taylortom I can add these is you want?

File: frontend/src/core/dashboard/dashboard.js (hunk 1 above)
Comment (taylortom) [inline frontend/src/core/dashboard/dashboard.js]: Done

**Comment by taylortom** [comment]
@dancgray have added changes based on above feedback

**Comment by dancgray** [comment]
Sorry @taylortom getting an error when editing a menu:

Uncaught TypeError: Cannot read property 'get' of undefined  editorRouter.js:167

**Comment by taylortom** [comment]
Closing this one for now

**Pull request closed without merging**